concent_api/core/views.py

```diff
@@ -586,3 +719,20 @@ def get_file_status(file_transfer_token_from_database: message.concents.FileTran
         return False
     else:
         raise exceptions.UnexpectedResponse()
+
+
+def decode_key(key: str) -> bytes:
+    return b64decode(key.encode('ascii'), validate = True)
+
+
+def decode_client_public_key(request):
+    return decode_key(request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY'])
+
+
+def decode_other_party_public_key(request):
+    if 'HTTP_CONCENT_OTHER_PARTY_PUBLIC_KEY' not in request.META:
```

Comment:
This function should return `None` explicitly if the condition here is false.

Reply:
If condition here is false it will either return decoded key or raise exception, there is no other exit from this function.

Reply:
OK. Sorry I missed that. It's OK then.